include/gsl/gsl_byte

```diff
@@ -153,6 +166,7 @@ constexpr byte to_byte_impl(T t) noexcept
     return static_cast<byte>(t);
 }
 template <>
+// GSL_SUPPRESS(type.4) // NO-FORMAT: attribute // TODO: suppression does not work
```

Comment:
I can't remember, but is there a reason this can't be just `return {t};`? If there is, perhaps we should leave behind a comment to explain to others reading the code.

Reply:
The only reason is that it does not work with c++14... will add comment.